modules/test/playwright/pages/commerce/commerce-product-content-web/productDetailsPage.ts

```diff
@@ -183,6 +184,7 @@ export class ProductDetailsPage {
 		this.priceField = async (price: string, container = this.page) => {
 			return container.getByText(price);
 		};
+		this.productDetails = page.locator('.product-detail');
```

Comment:
I think that this locator is too bound to the test. Either we remove it or we move the logic in the `quantitySelector` locator